pendulum_control/CMakeLists.txt

```diff
@@ -66,6 +66,59 @@ install(
 if(AMENT_ENABLE_TESTING)
   find_package(ament_lint_auto REQUIRED)
   ament_lint_auto_find_test_dependencies()
+
+  if(NOT WIN32)
```

Comment:
This if statement is unnecessary because the Cmake file already exits if WIN32 at the top of the file.